src/IBM.WatsonDeveloperCloud/Http/WatsonHttpClient.cs

```diff
@@ -42,6 +42,7 @@ public WatsonHttpClient(string baseUri)
                 this.BaseClient.BaseAddress = new Uri(baseUri);
 
             this.Formatters = new MediaTypeFormatterCollection();
+            this.Formatters.JsonFormatter.SupportedMediaTypes.Add(new MediaTypeHeaderValue("application/octet-stream"));

```

Comment:
@mediumTaj  Formatters is built on the three constructors, what do you think of creating a method in the IRequest / Request to add? So we can add it whenever necessary.